### PR TITLE
samples: shell: Remove setting LDSW active discharge at init.

### PR DESCRIPTION
**Pull request by magp-nordic**

### CODE REVIEW

File: samples/shell/src/main.c [hunk 1]
```diff
@@ -34,9 +34,6 @@ void main(void)
 
 	npmx_instance_t *npmx_instance = npmx_driver_instance_get(pmic_dev);
```
Comment (masz-nordic) [inline samples/shell/src/main.c:35]: Can be deleted as well?

Comment (magp-nordic) [reply samples/shell/src/main.c]: Removed

**Review by masz-nordic — APPROVED** [review]

File: samples/shell/src/main.c (hunk 1 above)
Comment (magp-nordic) [reply samples/shell/src/main.c:35]: Restored as previously branch was based on main, not on develop branch. Now this line cannot be removed, npmx_instance is used below.

**Review by masz-nordic — APPROVED** [review]

**Review by wilu-nordic — APPROVED** [review]

**Comment by nordic-segl** [comment]
All green:
https://jenkins-ncs.nordicsemi.no/job/HCS/job/NPMX/job/npmx_zephyr_develop_samples/view/change-requests/job/PR-203/

**Review by nordic-segl — APPROVED** [review]

**Pull request merged**